cluster/gce/util.sh

```diff
@@ -53,6 +53,17 @@ function join_csv {
 
 # Verify prereqs
 function verify-prereqs {
+  if [[ "${ENABLE_EXPERIMENTAL_API}" == "true" ]]; then
```

Comment:
We can easily check if we have explicit false. Nonetheless I don't think we need to do it now. I think it's worth a TODO, because if user sets RUNTIME_CONFIG enabling autoscaler will simply not work.